src/query/service/src/servers/http/v1/query/http_query_manager.rs

```diff
@@ -230,19 +226,16 @@ impl HttpQueryManager {
     }
 
     #[async_backtrace::framed]
-    pub(crate) async fn get_session(self: &Arc<Self>, session_id: &str) -> Option<Arc<Session>> {
-        let sessions = self.sessions.lock();
-        sessions.get(session_id)
-    }
-
-    #[async_backtrace::framed]
-    pub(crate) async fn add_session(self: &Arc<Self>, session: Arc<Session>, timeout: Duration) {
-        let mut sessions = self.sessions.lock();
-        sessions.insert(session.get_id(), session, Some(timeout));
-    }
-
-    pub(crate) fn kill_session(self: &Arc<Self>, session_id: &str) {
-        let mut sessions = self.sessions.lock();
-        sessions.remove(session_id);
+    pub(crate) fn get_token_manager(
+        self: &Arc<Self>,
+        last_query_id: &str,
+    ) -> Option<TxnManagerRef> {
+        let mut txn_managers = self.txn_managers.lock();
+        if let Some((txn_mgr, task_handle)) = txn_managers.remove(last_query_id) {
+            task_handle.abort();
+            Some(txn_mgr)
+        } else {
+            None
+        }
```

Comment:
why does getting `token manager` will mutate `txn_managers`?

Reply:
thanks, forgot to remove this code.